lib/order.rb

```diff
@@ -0,0 +1,49 @@
+require 'csv'
+# Order is responsible for tracking orders with prices for all orders from CSV
+class Order
+  attr_accessor :products, :customer, :fulfillment_status
+  attr_reader :id
+
+  # Creates one order record
+  def initialize(id, products, customer, fulfillment_status = :pending)
+    @id = id
+    @products = products
+    @customer = customer
+    @fulfillment_status = fulfillment_status
+    fs_types = %i[pending paid processing shipped complete]
+    raise ArgumentError, 'Wrong status' if fs_types.none? @fulfillment_status
+  end
+
+  # Calculates total price for one order with tax
+  def total
+    (@products.values.sum * 1.075).round(2)
+  end
+
+  def add_product(name, price)
+    raise ArgumentError if @products.key?(name)
```

Comment:
It might be better to provide a message to go with the `ArgumentError`, like:

```ruby
raise ArgumentError, "This order already has product: #{name}" if @products.key?(name)
```